.changeset/twelve-tigers-stare.md

```diff
@@ -0,0 +1,274 @@
+
+The following wallets currently support `rdns` configuration with RainbowKit:
+
+- Rainbow
+- MetaMask
+- Trust Wallet
+- Coinbase Wallet
+- Brave Wallet
+- Enkrypt Wallet
+- Frontier Wallet
+- OKX Wallet
+- Phantom
+- Rabby Wallet
+- Talisman
+- TokenPocket
+- XDEFI Wallet
+- Zerion
+
+**Disabling EIP6963 Support in Your dApp**
+
+If you don't want to support EIP6963 in your dApp, simply set `multiInjectedProviderDiscovery: false` in the `createConfig` function from wagmi.
+
+```tsx
+const wagmiConfig = createConfig({
+  chains,
+  connectors,
+  multiInjectedProviderDiscovery: false, // <- disable it here
+  transports: {
+    [mainnet.id]: http(),
+  },
+});
+```
+
+**EIP6963 Not Supported on Mobile**
+
+Please note that EIP6963 is not supported on mobile devices. Instead, we use the wallets provided to `connectorsForWallets` and/or `getDefaultWallets` and generate a WalletConnect URI to redirect you to the wallet app.
+
+**Rainbow Button Updates**
+
+The Rainbow Button package has been updated. We have transitioned from using the connector as a class to function.
+
+**Before:**
+
+```tsx
+import "@rainbow-me/rainbow-button/styles.css";
+import { configureChains, createConfig, WagmiConfig } from "wagmi";
+import { mainnet } from "wagmi/chains";
+import { publicProvider } from "wagmi/providers/public";
+import {
+  RainbowButtonProvider,
+  RainbowConnector,
+} from "@rainbow-me/rainbow-button";
+
+const { chains, publicClient, webSocketPublicClient } = configureChains(
+  [mainnet],
+  [publicProvider()]
+);
+
+const projectId = "YOUR_PROJECT_ID";
+
+const wagmiConfig = createConfig({
+  autoConnect: true,
+  connectors: [new RainbowConnector({ chains, projectId })],
+  publicClient,
+  webSocketPublicClient,
+});
+
+const App = () => {
+  return (
+    <WagmiConfig config={wagmiConfig}>
+      <RainbowButtonProvider>{/* Your App */}</RainbowButtonProvider>
+    </WagmiConfig>
+  );
+};
+```
+
+**After:**
+
+```tsx
+import "@rainbow-me/rainbow-button/styles.css";
+import { createConfig, http, WagmiProvider } from "wagmi";
+import { mainnet } from "wagmi/chains";
+import {
+  RainbowButtonProvider,
+  rainbowConnector,
+} from "@rainbow-me/rainbow-button";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+const projectId = "YOUR_PROJECT_ID";
+
+const wagmiConfig = createConfig({
+  connectors: [rainbowConnector({ projectId })],
+  chains: [mainnet],
+  transports: {
+    [mainnet.id]: http(),
+  },
+});
+
+const client = new QueryClient();
+
+const App = () => {
+  return (
+    <WagmiProvider config={wagmiConfig}>
+      <QueryClientProvider client={client}>
+        <RainbowButtonProvider>{/* Your App */}</RainbowButtonProvider>
+      </QueryClientProvider>
+    </WagmiProvider>
+  );
+};
+```
+
+**Wallet Button Updates**
```

Comment:
Are there any breaking changes here? It seems like it might not be necessary to explain